frontend/src/test/scala/bloop/bsp/BspProtocolSpec.scala

```diff
@@ -284,18 +284,15 @@ class BspProtocolSpec(
   test("find test classes") {
     TestUtil.withinWorkspace { workspace =>
       val logger = new RecordingLogger(ansiCodesSupported = false)
-      loadBspBuildFromResources("cross-test-build-scalajs-0.6", workspace, logger) { build =>
+      loadBspBuildFromResources("cross-test-build-scalajs-1.x", workspace, logger) { build =>
```

Comment:
My assumption is that the older version of Scala JS project might be more flaky than the 1.x version. We can see if it continuous down the road.